getting_started/step_by_step/signals.rst

```diff
@@ -343,6 +436,18 @@ To emit a signal in your scripts, call ``emit_signal()``.
         if health <= 0:
             emit_signal("health_depleted")
 
+ .. code-tab:: csharp C#
+
+    using Godot;
+
+    public partial class MyNode2D : Node2D
+    {
+        [Signal]
+        public delegate void HealthDepletedEventHandler();
```

Comment:
We don't use the `EventHandler` suffix in 3.x, because we use the name of the delegate as the name of the signal.

```suggestion
        public delegate void HealthDepleted();
```

But this example is in the wrong place, the GDScript example ends at line 419.